modules/basics/src/main/java/com/opengamma/strata/basics/ReferenceData.java

```diff
@@ -47,7 +48,22 @@ public static ReferenceData of(Map<? extends ReferenceDataId<?>, ?> values) {
    * @return standard reference data
    */
   public static ReferenceData standard() {
-    return StandardReferenceData.INSTANCE;
+    return StandardReferenceData.STANDARD;
+  }
+
+  /**
+   * Obtains the minimal set of reference data.
+   * <p>
+   * The {@linkplain #standard() standard} reference data contains common holiday calendars
+   * and indices, but may not be suitable for production use. The minimal reference data contains
+   * just those identifiers that are needed by Strata, and that are non-controversial.
+   * These are {@link HolidayCalendars#NO_HOLIDAYS}, {@link HolidayCalendars#SAT_SUN},
+   * {@link HolidayCalendars#FRI_SAT} and {@link HolidayCalendars#THU_FRI}.
+   *
+   * @return minimal reference data
+   */
+  public static ReferenceData minimal() {
```

Comment:
I didn't really want to do this as it is a bit weird behaviour, but since the system doesn't work without them, accepting this nastiness will probably avoid some user bugs.